docs/changehistory/NextVersion.md

```diff
@@ -289,6 +289,12 @@ Previously, nested related properties of different intermediate classes were all
 | ---------------------------------------------------------------------------------------------- | ----------------------------------------------------------------------------------------------------------- |
 | ![All properties fall under one category](./media/AllSpatialCategoriesUnderPhysicalObject.png) | ![Properties fall under different categories](./media/SpatialCategoriesUnderPhysicalObjectAndTestClass.png) |
 
+### Relationship properties
+
+It is now possible to include relationship properties into content using a newly added [`RelatedPropertiesSpecification.relationshipProperties`](../presentation/content/RelatedPropertiesSpecification.md#attribute-relationshipproperties) attribute. Properties of the relationship and category of the target class are categorized under relationship category if any relationship properties are included.
+
+It's now also possible to move related properties under a relationship category even when there are no relationship properties. To force this behavior, the new [`RelatedPropertiesSpecification.forceCreateRelationshipProperties`](../presentation/content/RelatedPropertiesSpecification.md#attribute-forcecreaterelationshipcategory) attribute can be used.
```

Comment:
> It's now also possible to move related properties under a relationship category even when there are no relationship properties.

Does this mean that relationship category will not appear even if contains properties from foreign sources? If so, this should be clearly explained in the documentation.

Reply:
What exactly do you mean by "foreign sources"?

A ruleset that moves related properties under a relationship category:
```json
{
  "ruleType": "Content",
  "specifications": [{
      "specType": "SelectedNodeInstances",
      "relatedProperties": [{
          "propertiesSource": [{
              "relationship": {"schemaName": "BisCore", "className": "ModelModelsElement"},
              "direction": "Forward",
              "targetClass": {"schemaName": "BisCore", "className": "PhysicalPartition"}
            }],
          "properties": ["UserLabel"],
          "forceCreateRelationshipCategory": true
        }]
    }]
}
```

Reply:
Thanks, I now have a better understanding of this new feature.